src/components/WwwFrame/TheHeader.vue

```diff
@@ -797,6 +801,11 @@ export default {
 	@apply tw-h-8 md:tw-h-9 tw-whitespace-nowrap tw-flex-shrink-0;
 }
 
+.header__login__button {
+	background-color: #fff;
```

Comment:
Is this necessary? And if it is, couldn't we use a tailwind class?

Reply:
Is there any example with that class, I haven't found it yet 😞

Reply:
`tw-bg-white`